changelog.d/1161.misc

```diff
@@ -0,0 +1 @@
+Bridge IRC `reason`s when users QUIT or PART
```

Comment:
Due to accidental dependency on #1160. Diffs should look more sensible once that one's been merged.